.dockerignore

```diff
@@ -42,6 +42,9 @@ pids
 **/.coverage
 **/.coverage.*
 
+# Debug directory generated by tools like Cargo
+**/debug/
+
```

Comment:
Never seem a `debug` folder, let's not add it.

Reply:
Information about when a `debug` dir may be generated: https://nnethercote.github.io/perf-book/build-configuration.html I'll remove this, but IMO it is unlikely we would ever want it committed as part of source.

Reply:
Addressed in https://github.com/hashintel/hash/pull/2938/commits/211e32f469a1e5fe75e195250422ebd111c781af